utils/checkWalletBlacklist.ts

```diff
@@ -0,0 +1,16 @@
+import { getWalletTransactions } from "clients/moralis";
+
+export const checkWalletBlacklist = async (walletAddress: string) => {
+    const walletTransactions = await getWalletTransactions(walletAddress);
+
+    // Update list with blacklisted addresses
+    const blacklistedAddresses = ["ADDRESS1", "ADDRESS2", "ADDRESS3"];
```

Comment:
This is not exactly what I meant.[ Check this ](https://www.moleculeprotocol.io/) this is the idea. 
Where there should be some tool existing that will check if connected address has communicated with bad actor